plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/AwsSdkUtil.java

```diff
@@ -31,10 +36,48 @@ private AwsSdkUtil() {}
     /**
      * Helper method to get all results from a paginated API.
      *
-     * @param request request object reused for subsequent requests with
+     * @param builder builder object reused for subsequent requests with
      * {@code setNextToken} being used to set the next token in the request object
      */
-    public static <Request, Result> Stream<Result> getPaginatedResults(
+    public static <Request, Result, Builder> Stream<Result> getPaginatedResults(
```

Comment:
We should be able to use methods like `getDatabasesPaginator()` so that the SDK handles pagination. Then we can remove this method.

Reply:
done